ci/matrix.xml

```diff
@@ -80,53 +80,53 @@
         ++$scala noIrCheckTest/test \
         noIrCheckTest/clean &&
     sbtretry 'set scalaJSOptimizerOptions in testSuite ~= (_.withDisableOptimizer(true))' \
-        ++$scala testSuite/test &&
+        ++$scala testSuite/test scalaTestSuite/test &&
```

Comment:
You also need to apply the `scalaJSOptimizerOptions` to `scalaTestSuite`, otherwise this is the same as above. The same comment applies to almost all lines from here.